src/events/event.controller.ts

```diff
@@ -0,0 +1,16 @@
+import { Request, Response } from 'express';
+import { EventHandler } from './event.handler';
+
+export class EventController {
```

Comment:
Write a line or two of documentation before every class and function/method (what id does, explanation about each parameter and the return value

Reply:
done